setup.py

```diff
@@ -9,7 +9,7 @@
 
 
 NAME = "pbbt"
-VERSION = "0.1.6"
+VERSION = "0.1.7"
```

Comment:
should be 0.2.0, I think, as we are dropping Python 2 support here

Reply:
Makes sense, updated the PR.